azure-pipelines.yml

```diff
@@ -0,0 +1,83 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+      - release/*
+
+resources:
+  repositories:
+    - repository: niveristand-custom-device-build-tools
+      type:       github
+      ref:        main
+      endpoint:   nivs-custom-devices
+      name:       ni/niveristand-custom-device-build-tools
+
+stages:
+  - template: azure-templates/stages.yml@niveristand-custom-device-build-tools
+    parameters:
+      lvVersionsToBuild:
+        - version: '2020'
+          bitness: '32bit'
+        - version: '2021'
+          bitness: '64bit'
+        - version: '2023'
+          bitness: '64bit'
+
+      buildSteps:
+        - projectLocation: 'Source\Core\Data Sharing Framework Core Build - Windows.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Data Sharing Framework Core'
+
+        - projectLocation: 'Source\Core\Data Sharing Framework Core Build - Linux RT.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'Linux RT x64'
+          buildSpec: 'Data Sharing Framework Core'
+
+        - projectLocation: 'Source\Custom Device\Data Sharing Framework Custom Device.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Configuration Release'
+
+        - projectLocation: 'Source\Custom Device\Data Sharing Framework Custom Device.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Scripting API'
+
+        - projectLocation: 'Source\Scripting Examples\Data Sharing Framework Scripting.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Scripting Examples'
+
+        - projectLocation: 'Source\Custom Device\Data Sharing Framework Custom Device.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Engine Release'
+
+        - projectLocation: 'Source\Core\Data Sharing Framework Core Build - Linux RT.lvproj'
```

Comment:
Needs to point to the custom device project: `Source\Custom Device\Data Sharing Framework Custom Device Linux RT x64.lvproj`

Reply:
01e9e359ff20174dcff22eaafbeeacf29e12c076